rskj-core/src/main/java/co/rsk/net/BlockSyncService.java

```diff
@@ -73,7 +73,13 @@ public BlockSyncService(
         this.blockHeaderValidator = blockHeaderValidator;
     }
 
-    protected boolean preprocessBlock(@Nonnull Block block, Peer sender, boolean ignoreMissingHashes) {
+    /**
+     * Does initial preprocessing of the {@code block}.
+     * 
+     * @return block and its ancestors, if any. Returns an empty list, if the block is too advanced, already in a local chain
```

Comment:
_@return block and its ancestors, if any_

Shouldn't this be sth like _@return block and its ancestors that are not yet in blockchain_?